### PR TITLE
feat: Remove Argo-Events

### PR DESCRIPTION
**Pull request by tumido**
Remove Argo Events from `dh-*-argo`

Related: https://github.com/AICoE/aicoe-cd/pull/276 https://github.com/AICoE/idh-manifests/pull/145

### CODE REVIEW

File: manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml
Comment (HumairAK) [inline manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml:19]: did you also mean to remove this flag?

Comment (tumido) [reply manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml:19]: I don't think it's removing it.. it's the indentation.. I'll remove the change.

**Review by anishasthana — COMMENTED** [review]
/hold

**Comment by tumido** [comment]
@anishasthana it's already on hold, don't worry :smile: https://github.com/AICoE/aicoe-cd/pull/277#issuecomment-829303767

**Comment by anishasthana** [comment]
Heh, apparently I was very tired and didn't see the label. I was so confused as to why Sesheta was ignoring me :-)

**Review by anishasthana — COMMENTED** [review]
With https://gitlab.cee.redhat.com/data-hub/s3-backup-util/-/merge_requests/9, I think we are good to merge these changes in when we want. Lets take a look at this PR again on Monday/Tuesday -- if the cron workflow works correctly internally, we should be good to go

**Review by gmfrasca — COMMENTED** [review]
/lgtm

**Review by anishasthana — APPROVED** [review]
/unhold

**Pull request merged**
